src/Control/Email/Email.php

```diff
@@ -584,13 +591,16 @@ public function setPriority($priority)
     }
 
     /**
+     * @deprecated 4.12.0 Will be replaced with Symfony\Component\Mime\Email::attachFromPath()
+     *
      * @param string $path Path to file
      * @param string $alias An override for the name of the file
      * @param string $mime The mime type for the attachment
      * @return $this
      */
     public function addAttachment($path, $alias = null, $mime = null)
     {
+        Deprecation::notice('4.12.0', 'Will be replaced with Symfony\Component\Mime\Email::attachFromPath()');
```

Comment:
```suggestion
        Deprecation::notice('4.12.0', 'Use attachFromPath() instead');
```

Reply:
The CMS 5 PR still has this method. Are we removing it or not?

Reply:
Not removing, keeping alias, not deprecating, will update PR